### ISSUE TITLE
Write documentation

### ISSUE THREAD

**Issue by fallen** [pytition/Pytition#218]
Documentation should at least address the following points:

* Installation
* Configuration
* Usage
* Update
* Backup

**Comment by martinlehoux** [comment]
Have you started yet to work on this ? Is there already the documentation architecture ?

**Comment by fallen** [comment]
Yes, documentation is in `doc/` , it's in sphinx format (.rst), it can easily be compiled with `make html` from the doc directory.
It's online at https://pytition.readthedocs.io/

**Comment by fallen** [comment]
So far it's only an Installation documentation and a start of Configuration doc'.

**Comment by martinlehoux** [comment]
I would like to work on the backup doc. When talking about backup, is it just about a db backup + media backup ? Is there any technology we want to advise using ?

**Comment by fallen** [comment]
> I would like to work on the backup doc. When talking about backup, is it just about a db backup + media backup ? Is there any technology we want to advise using ?

That's indeed an important point.
I have no technology in mind yet. Feel free to do some research about what's best. Maybe the Django project has some standard/advised tooling?

But in the end I would also say that it's boils down to backuping:
* db
* release version (or SHA-1 ? or entire source tree ?)
* config
* media

**Comment by martinlehoux** [comment]
I looked into Django docs, there is hardly anything about how to make backups.
There is this repo : https://github.com/django-dbbackup/django-dbbackup which appears in several search results, managing database and media files, maybe i'll take a look at it

**Comment by fallen** [comment]
Update and backup documentation is now done: https://pytition.readthedocs.io/en/latest/update.html
